ddtrace/__init__.py

```diff
@@ -6,6 +7,10 @@
 
 __version__ = '0.12.1'
 
+# configure the root logger
+logging.basicConfig()
```

Comment:
Configuring *where* the logs go from a library/module (rather than the application) is generally frowned upon: 
https://www.loggly.com/ultimate-guide/python-logging-basics/#logging-from
and https://sweetness.hmmz.org/2013-11-18-use-of-logging-package-from-within-a-library.html (the `Explicitly configuring handlers` section)

Reply:
This change has been reverted in this PR: https://github.com/DataDog/dd-trace-py/pull/556

A new 0.13.1 release is scheduled for today. Thanks for the heads-up!